app/services/doc_auth/mock/result_response.rb

```diff
@@ -166,15 +183,23 @@ def classification_info
       end
 
       def doc_auth_result_from_success
-        if success?
+        if doc_auth_success?
           DocAuth::LexisNexis::ResultCodes::PASSED.name
         else
           DocAuth::LexisNexis::ResultCodes::CAUTION.name
         end
       end
 
-      def all_doc_capture_values_passing?(doc_auth_result, id_type_supported)
-        doc_auth_result == 'Passed' &&
+      def transaction_status_from_success
```

Comment:
this name is confusing to me - I'm not sure I understand the `from_success` piece.

Reply:
continuing the pattern used in `doc_auth_result_from_success` this is if the document is authenticated successfully